code/src/components/Album.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+import { Artist } from "./Artist";
+import { Icons } from "./Icons";
+
+export const Album = (props) => {
+  return (
+    <div className="album">
+      <div className="album-image">
+        {props.album.images
+          .filter((data) => {
+            if (data.width === 640) {
+              //if filter is true then it keeps the image and goes to the .map
+              return true;
+            }
+            // all false filters are removed and are not passed to the .map
+            return false;
+          })
+          .map((data) => {
+            // console.log(data);
+            return (
```

Comment:
This was an interesting solution, with the if before moving on to ` map()` could you explain more about it?
/ Ida & Emma